packages/shared/routes/setup/migrate/views/TransferFragmentedFunds.svelte

```diff
@@ -232,50 +235,49 @@
                                               return _transaction
                                           })
                                           const reverseTrytesLedger = trytes.reverse()
-                                          prepareMigrationLog(bundleHash, reverseTrytesLedger, transaction.balance)
+                                          prepareMigrationLog(reverseTrytesLedger, transaction.balance, bundleHash)
                                           return sendOffLedgerMigrationRequest(reverseTrytesLedger, transaction.index)
                                       })
                                       .then((receipt) => {
-                                          migrationLog.update((_migrationLog) => [
-                                              ..._migrationLog,
-                                              (_migrationLog[idx] = {
-                                                  ..._migrationLog[idx],
-                                                  requestId: receipt?.request?.requestId || '',
-                                              }),
-                                          ])
+                                          updateRequestInMigrationLog(receipt?.request, idx)
```

Comment:
If user retries the migration new log is added on this line `prepareMigrationLog(reverseTrytesLedger, migratableBalance, bundleHash)` but  `updateRequestInMigrationLog(receipt?.request, idx)` always updates the errorMessage for the idx.